### PR TITLE
First pass at new readme.

### PR DESCRIPTION
**Pull request by amouat**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,66 +1,110 @@
...
 
-## Why
+apko has the following key features:
+
+ - **Fully reproducible by default.** Run apko twice and you will get exactly the same binary.
+ - **Fast.** apko aims to build images in ms.
+ - **Small.** apko generated images only contain what's needed by the application,
+   in the style of [distroless](https://github.com/GoogleContainerTools/distroless)
+ - **SBOM Support.** apko produces a Software Bill of Materials (SBOM) for images, detailing all the packages inside.
+ - **Services.** apko supports using the s6 supervisor tool to run multiple processes
```
Comment (kaniini) [inline README.md:12]:
```suggestion
 - **Services.** apko supports using the [s6 supervision suite](https://skarnet.org/software/s6) to run multiple processes
```

File: README.md [hunk 2]
```diff
@@ -1,66 +1,110 @@
...
+```
+$ docker load < apko-alpine.tar
+Loaded image: apko-alpine:test
+$ docker run -it apko-alpine:test
+e289dc84c4ad:/# echo boo!
+boo!
+```
 
-    # apko build config.yaml tag output.tar
+Or publish the image directly to a registy.
```
Comment (kaniini) [inline README.md:57]:
```suggestion
Or publish the image directly to a registry.
```

File: README.md [hunk 3]
```diff
@@ -1,66 +1,110 @@
...
+The [melange project](https://github.com/chainguard-dev/melange) is designed to produce apk packages to be used in apko.
 
-### SBOMs
+The [ko](https://github.com/google/ko) project builds Go projects from source in a similar manner to apko.
 
-As a result of using apk to manage distribution and build artifacts,
-we are able to generate SBOMs for containers.  Future versions will
-have even better SBOMs, using the apk-tools 3.x SBOM feature.
+The [kontain.me](https://github.com/imjasonh/kontain.me) service creates fresh container images on
+demand using differnt forms of declarative configuration (including ko and apko).
```
Comment (kaniini) [inline README.md:110]:
```suggestion
demand using different forms of declarative configuration (including ko and apko).
```

**Review by kaniini — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (amouat) [reply README.md]: Done (didn't use the GH tool as I'd already pushed a change).

**Pull request merged**